styles/portfolio.css

```diff
@@ -0,0 +1,191 @@
+body{
```

Comment:
in this file you can remove the white space in between each code block and also remove the commented code